zebra/zebra_dplane.c

```diff
@@ -1108,6 +1136,21 @@ vrf_id_t dplane_ctx_get_vrf(const struct zebra_dplane_ctx *ctx)
 	return ctx->zd_vrf_id;
 }
 
+/* In some paths we have only a namespace id */
+void dplane_ctx_set_ns_id(struct zebra_dplane_ctx *ctx, ns_id_t nsid)
```

Comment:
Just a suggestion. Can all this be converted to 'static inline's and moved to some .h file instead?

Reply:
not really - the definition of the context struct itself is opaque, so callers can't see the internal details. if we had c++, we'd do inlines, of course, but ...

Reply:
I see. But that does not seem to be a good practice t me. Anyways it's fine.